README.md

```diff
@@ -1,96 +1,100 @@
+* `library_name`: Name of the library for which you are creating an API directory. For example, the library name could be ``core``, ``file-transfer``, or an empty string.
+* `project_name_slug`: Name for the API package. In most cases, the suggested default should work fine.
+* `api_version`: Version number of the API package. If you're just getting started, ``0`` should be used.
+* `api_package_version`: Version of the API package. If this is the first release, the default ``0.1.0`` is suitable.
+* `protos_dir`: Path to a directory that contains the PROTO files to include in the API package. This can be left empty, in which case must manually copy in the PROTO files later.
 
-* `product_name`: The name of the product for which you are creating an API directory. For example, the product name could be `MAPDL`, `DPF`, or `tools`.
-* `library_name`: The name of the library for which you are creating an API directory. For example, the library name could be `core`, `file-transfer` or an empty string.
-* `project_name_slug`: The package name used for the API package. In most cases, the suggested default should work fine.
-* `api_version`: Version number of the API. If you're just getting started, `0` should be used.
-* `api_package_version`: The version of the API package. The default `0.1.0` is suitable if this is the first release.
-* `protos_dir`: Path to a directory which contains the `.proto` files that should be included in the API package. This can be left empty, in which case you need to manually copy in the `.proto` files later.
+  **NOTE:** The path must be given relative to the root of the package that is to be created. In other words, you should add a leading ``../`` compared path relative to your working directory, or alternatively use an absolute path.
+* `proto_dependencies`: Specifies whether your PROTO files depend on another API package. If they are independent, you can accept the default. Otherwise, you must pass dependencies as a JSON dictionary:
 
-  **NOTE:** The path needs to be given relative to the root of the package that will be created. In other words, you should add a leading `../` compared path relative to your working directory, or alternatively use an absolute path.
-* `proto_dependencies`: Specifies whether your `.proto` files depend on another API package. If they are independent, you can simply accept the default. Otherwise, the dependencies need to be passed as a JSON dictionary:
   ```json
   {"modules": [<your dependencies here>]}
   ```
 
-  For example, if you depend on ``ansys-api-file-transfer`` and ``ansys-api-mapdl``:
+  For example, if you depend on the ``ansys-api-file-transfer`` and ``ansys-api-mapdl`` packages, you
+  would pass these dependencies:
+
   ```json
   {"modules": ["ansys-api-tools-file-transfer", "ansys-api-mapdl-core"]}
   ```
 
-## Maintaining your API repository
+## API repository maintenance
```

Comment:
This section refers to your API repository, I think we should keep the section title as-is.

```suggestion
## Maintaining your API repository
```